### PR TITLE
Remove lib/helpers.js

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - Remove unnecessary methods from lib/helpers.js

### CODE REVIEW

File: samples/js/node/common_api_example.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ async function run()
   const output = await model.infer(tensor, shape);
 
   //show the results
-  console.log('Result: ' + imagenetClassesMap[math.argMax(output.data)]);
-  console.log(math.argMax(output.data));
+  const result = getMaxElement(output.data);
+  console.log('Result: ' + imagenetClassesMap[result.index],
```
Comment (vishniakov-nikolai) [inline samples/js/node/common_api_example.js:34]: Is it possible to use `printClass` function from `./samples/common/helpers.mjs` here?

File: samples/js/node/ppp_example.js [hunk 2]
```diff
@@ -38,6 +37,7 @@ async function run()
   const output = model.compile('CPU').infer(tensor);
 
   //show the results
-  console.log('Result: ' + imagenetClassesMap[math.argMax(output.data)]);
-  console.log(math.argMax(output.data));
+  const result = getMaxElement(output.data);
+  console.log('Result: ' + imagenetClassesMap[result.index],
```
Comment (vishniakov-nikolai) [inline samples/js/node/ppp_example.js:41]: Same question here

File: samples/js/node/read_model_async_example.js [hunk 3]
```diff
@@ -23,9 +23,9 @@ async function run()
   Promise.all([modelPromise, tensorPromise]).then(([model, tensor]) => {
     const output = model.compile('CPU').infer(tensor);
     //show the results
-    console.log('Result: ' + imagenetClassesMap[math.argMax(output.data)]);
-    console.log(math.argMax(output.data));
-  });
+    const result = getMaxElement(output.data);
+    console.log('Result: ' + imagenetClassesMap[result.index],
```
Comment (vishniakov-nikolai) [inline samples/js/node/read_model_async_example.js:27]: And here

**Review by vishniakov-nikolai — APPROVED** [review]
If you can use `printClass` function in your examples it would be great.
The rest is fine.

File: samples/js/node/common_api_example.js (hunk 1 above)
Comment (almilosz) [reply samples/js/node/common_api_example.js:33]: It is possible, but this function generates a warning that importing JSON modules is an experimental feature and might change at any time.
I propose to use this function here after we change it.

**Pull request merged**